swap/peer.go

```diff
@@ -184,7 +177,6 @@ func verifyChequeAgainstLast(cheque *Cheque, lastCheque *Cheque, expectedAmount
 		actualAmount -= lastCheque.Amount
 	}
 
-	// TODO: maybe allow some range around expectedAmount?
```

Comment:
Removed for now. I would like to create an issue that proposes to remove the honey amount at all in the cheque.